posts/2021/12/conda-recipe-grayskull.md

```diff
@@ -0,0 +1,124 @@
+<!--
+.. title: On Software Packages, Conda and Recipes
+.. slug: conda-recipe-grayskull
+.. date: 2021-12-08 10:00:00 UTC+00:00
+.. author: Mahe Iram Khan
+.. tags: Conda, conda-forge, recipes, Grayskull, packaging, Python
+.. category:
+.. link:
+.. description:
+.. type: text
+.. previewimage: /images/2021/12/grayskull.jpg
+-->
+
+Python might be the most popular snake out there, but most of us have also heard of that other serpent: Conda. And some of us have wondered what it really is. In this post we’ll learn about Conda, software packages and package recipes. Most importantly we’ll learn about Grayskull — a conda recipe generator.
+
+<!-- TEASER_END -->
+
+Hey! I’m [Mahe](https://twitter.com/IramMahe), a Computer Engineering student from India.
+During the summer of 2021 I was an intern at Quansight Labs and I worked on a project called ‘Grayskull’.
+Before we learn about Grayskull, as promised, I’ll talk about software packages and Conda.
+
+## Software Packages
+A software package is an installable piece of code that somebody wrote and published for others to use. In other words, you can install packages so you can benefit from the code others have written without reinventing the wheel.
+
+```python
+import numpy
+```
+
+That’s you ‘importing’ the package numpy into your code.
+
+## Channels
+Once you have written a package you might want to publish it so that others can download it and use it. Depending on the packaging technology you are using, the online location where these packages are made available will receive a different name. In the Conda world, they are called _channels_.
+Channels are like warehouses of packages.
```

Comment:
Later on you talk about PyPI, so introduce it here
```suggestion
## Hosting packages
Once you have written a package you might want to publish it so that others can download it and use it. Depending on the packaging technology you are using, the online location where these packages are made available will receive a different name. Typically, users will know about hosting packages on [PyPI](https://pypi.org/). In the Conda world, packages are hosted on _channels_.
Channels are like warehouses of packages.
```